javascript/ql/src/semmle/javascript/Expr.qll

```diff
@@ -698,7 +983,16 @@ class CallExpr extends @callexpr, InvokeExpr {
   Expr getReceiver() { result = getCallee().(PropAccess).getBase() }
 }
 
-/** A method call expression. */
+/**
+ * A method call expression.
+ *
+ * Examples:
+ *
+ * ```
+ * Object.create(null)
+ * [1, 2, 3].forEach(console.log);
```

Comment:
Maybe replace `console.log` with a closure so people won't think we model the call to `console.log` as a `MethodCallExpr`.